src/withDirection.jsx

```diff
@@ -68,8 +71,7 @@ export default function withDirection(WrappedComponent) {
   WithDirection.contextTypes = contextTypes;
   WithDirection.displayName = `withDirection(${wrappedComponentName})`;
   if (WrappedComponent.propTypes) {
-    WithDirection.propTypes = deepmerge({}, WrappedComponent.propTypes);
-    delete WithDirection.propTypes.direction;
+    WithDirection.propTypes = deepmerge(WrappedComponent.propTypes, withDirectionPropTypes);
```

Comment:
it might be optional here, but it should stay required on the wrapped component.